src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder_HonorsTokensProperty.java

```diff
@@ -17,8 +17,8 @@
 package com.puppycrawl.tools.checkstyle.checks.imports.importorder;
 
 import static javax.swing.WindowConstants.DISPOSE_ON_CLOSE; // ok
-import static java.awt.Button.ABORT; // ok
-import java.awt.Dialog; // ok
+import static java.awt.Button.ABORT; // violation
+import java.awt.Dialog; // violation
```

Comment:
Please add violation message to comment.

Reply:
Also, we need to update config for this file.

Reply:
Please clarify how you want config file updated. I left it alone to show this is how the check will function for users who did use this weird configuration before this change.

Do you wish me to remove the tokens completely?

Reply:
Let's just remove `tokens`

Reply:
Done both.

Removed tokens from config header and added violation message to comment.